### DISCUSSION TITLE
How to Export report via email for hosts have security updates pending for install and available

### DISCUSSION THREAD

**Discussion by OpenThink1** [furlongm/patchman#592] [Q&A] [answered]
How to Export list of hosts have security updates pending via email.

**Comment by furlongm** [comment, marked as answer]
https://gist.github.com/furlongm/a194697b7aa171411b7fffe2a64b06cc

has an example that uses the api to get updates

**Reply by OpenThink1** [reply]
Many thanks Marcus

**Reply by OpenThink1** [reply]
Hello,
Would the scripts able to email them if below email/updatereport.sh please?

https://github.com/furlongm/patchman/pull/62/commits/def37198d34400e96d088d27daf2f2fd952f554a

Kind regards,
Fuad

**Comment by tabbitsp** [comment]
Hi,
actually, the script above exports infos on ALL updates - I did not find any clue on how to export the number of "security updates" only. Is there also a possibility for this?
